composer/algorithms/blurpool/blurpool_layers.py

```diff
@@ -220,16 +220,18 @@ class BlurConv2d(nn.Module):
 
     # based partially on https://pytorch.org/docs/stable/_modules/torch/nn/modules/conv.html#Conv2d
 
-    def __init__(self,
-                 in_channels: int,
-                 out_channels: int,
-                 kernel_size: _size_2_t,
-                 stride: _size_2_t = None,
-                 padding: _size_2_t = 0,
-                 dilation: _size_2_t = 1,
-                 groups: int = 1,
-                 bias: bool = True,
-                 blur_first: bool = True):
+    def __init__(
```

Comment:
Have we moved to this style of formatting? Is this done automatically?

Reply:
So... we're technically unopinionated on how we format these things. You can either have a trailing comma or not have it. I _personally_ prefer the the new format, and have some linters that do things like that for me. Must have snuck through, not removing since we're ok with both formats.